learning-objectives/data.yml

```diff
@@ -373,31 +350,34 @@ business-understanding:
   - goals-and-kpis
   - benchmark
 
-research:
+research: # used in projects design-and-usability
```

Comment:
@lisperaza various objetivos antigua sigue ser en el proyecto Design and Usability. Quitamos este proyecto o cambiamos/quitamos los objetivos dentro a los nuevos?